src/dynamics/fixed_points/basic.lean

```diff
@@ -76,6 +78,18 @@ lemma preimage_iterate {s : set α} (h : is_fixed_pt (set.preimage f) s) (n : 
   is_fixed_pt (set.preimage (f^[n])) s :=
 by { rw set.preimage_iterate_eq, exact h.iterate n, }
 
+protected lemma symm (h : is_fixed_pt e x) : is_fixed_pt e.symm x :=
+h.to_left_inverse e.left_inverse_symm
+
+protected lemma inv (h : is_fixed_pt e x) : is_fixed_pt ⇑(e⁻¹) x := h.symm
+
+protected lemma pow (h : is_fixed_pt e x) (n : ℕ) : is_fixed_pt ⇑(e ^ n) x :=
+by { rw ←equiv.perm.iterate_eq_pow, exact h.iterate _ }
+
+protected lemma zpow (h : is_fixed_pt e x) : ∀ n : ℤ, is_fixed_pt ⇑(e ^ n) x
+| (int.of_nat n) := h.pow _
+| (int.neg_succ_of_nat n) := (h.pow $ n + 1).inv
```

Comment:
```suggestion
protected lemma perm_symm (h : is_fixed_pt e x) : is_fixed_pt e.symm x :=
h.to_left_inverse e.left_inverse_symm

protected lemma perm_inv (h : is_fixed_pt e x) : is_fixed_pt ⇑(e⁻¹) x := h.perm_symm

protected lemma perm_pow (h : is_fixed_pt e x) (n : ℕ) : is_fixed_pt ⇑(e ^ n) x :=
by { rw ←equiv.perm.iterate_eq_pow, exact h.iterate _ }

protected lemma perm_zpow (h : is_fixed_pt e x) : ∀ n : ℤ, is_fixed_pt ⇑(e ^ n) x
| (int.of_nat n) := h.pow _
| (int.neg_succ_of_nat n) := (h.pow $ n + 1).perm_inv
```
since these lemmas are also true for `add_aut` etc.